### PR TITLE
Example showing similar companies

### PR DESCRIPTION
**Pull request by khellan**
Add similar companies example showing how to use this API from Python.

### CODE REVIEW

File: python/similar_companies/service.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+from sys import argv
+
+HEADERS = {
+    'Accept': 'application/json',
+    'x-api-key': API_KEY
+}
+
+
+class SimilarCompaniesService:
+    COMPANY = "https://openapi.companybooknetworking.com/1.0/similar-companies/company"
```
Comment (osblinnikov) [inline python/similar_companies/service.py]: version can be 1.2 now

Comment (osblinnikov) [reply python/similar_companies/service.py]: COMPANY const better be renamed to something like SIMILAR_COMPANIES_URL

**Review by osblinnikov — APPROVED** [review]
LGTM except a small comment

File: python/similar_companies/service.py (hunk 1 above)
Comment (khellan) [reply python/similar_companies/service.py]: The endpoint is company so I could rename it COMPANY_URL

**Pull request merged**